### PR TITLE
Remove prompt code examples

### PR DESCRIPTION
**Pull request by stripes416**
Addresses #86 -- prompt removal from code examples.
Other requests are addressed in PR #93

### CODE REVIEW

**Comment by stripes416** [comment]
I don't really know what the standard for the project is as far as multi-commit vs squashed PRs, but please let me know if I need to make those kind of changes.

File: getting_started/setup/setup_up_2_board.rst [hunk 1]
```diff
@@ -66,9 +66,9 @@ Copy kconfig fragments for SOF development from https://github.com/thesofproject
 
 .. code-block:: bash
 
-   $ make defconfig
-   $ scripts/kconfig/merge_config.sh .config <path>/kconfig/base-defconfig <path>/kconfig/sof-defconfig  <path>/kconfig/hdaudio-codecs-defconfig
-   $ make -j8
+   make defconfig
+   scripts/kconfig/merge_config.sh .config <path>/kconfig/base-defconfig <path>/kconfig/sof-defconfig  <path>/kconfig/hdaudio-codecs-defconfig
```
Comment (intelkevinputnam) [inline getting_started/setup/setup_up_2_board.rst]: Need to be careful about those angle brackets. Copy and pasting them wouldn't work out very well. OK for now, I guess, though in the long term I think a better solution is to tell the user to set a variable and use the variable in the instructions.

**Comment by xiulipan** [comment]
@stripes416 
Thanks for helping!

File: getting_started/setup/setup_up_2_board.rst (hunk 1 above)
Comment (stripes416) [reply getting_started/setup/setup_up_2_board.rst]: @intelkevinputnam I agree --- I think you referenced this solution in #86. I'll work on it and open a PR.

Comment (intelkevinputnam) [reply getting_started/setup/setup_up_2_board.rst]: No worries. I just like to beat dead horses.

**Pull request merged**